src/containers/MyComplaints/layout/index.js

```diff
@@ -1,18 +1,117 @@
-import {Text, View} from 'react-native';
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  Alert,
+  SafeAreaView,
+  FlatList,
+} from 'react-native';
+import React, {useState, useEffect, useRef} from 'react';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import {colors} from '../../../library/styles/vars';
+import {sortArrayByDate, dateFormatter} from '../../../library/methods';
+import {styles} from '../styles';
 
-import {PrimaryButton} from '../../../components';
-import React from 'react';
+import {DATA, MyComp, MyCompEmpty} from '../dummyData';
 
-const SendDataAdd = props => {
-  const next = () => {
-    props.navigation.navigate('TitleDescriptionAdd', {
-      ...props.route.params,
-    });
+const ComplaintItem = ({category, title, date, id}) => {
+  const onEditPress = () => {
+    //Edit View
   };
+  //Confirmation alert
+  const onDeletePress = () => {
+    Alert.alert(
+      '¿Borrar esta denuncia?',
+      `Tu denuncia <${title}> se eliminará de forma permanente.`,
+      [
+        {
+          text: 'Cancelar',
+        },
+        {
+          text: 'Sí, borrar',
+          onPress: () => {
+            //delete Complaint functionality
+          },
+        },
+      ],
+    );
+  };
```

Comment:
Move all the text in a constants file in order to internalization be easier to apply

Reply:
Done, this sort of messages are now retrieved from a different file located at library/constant directory.